tools/ci_build/build.py

```diff
@@ -872,6 +872,7 @@ def install_python_deps(numpy_version=""):
     dep_packages.append("sympy>=1.10")
     dep_packages.append("packaging")
     dep_packages.append("cerberus")
+    dep_packages.append("psutil")
```

Comment:
Going forward we should put all such things in a requiremnts.txt so that CG can recognize